cmake/syslib/arduino-esp32

```diff
@@ -0,0 +1 @@
+Subproject commit 7d7824701fe5e22f08555d3e1ce3180a922b2151
```

Comment:
Which commit is this? Can we specify a tag instead?

Reply:
where do you see that? There are no changes/commits to the arduino-esp32 subproject I am aware of.

Reply:
I was meaning the commit has, we can ignore this one for now. It will go away soon after v1.3.0 and I sort through the libs